src/TourManager/Repository/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
```

Comment:
Connection String is MySql , ef provider is MsSql and in docker compose you have defined postgreSql )))